llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -831,7 +843,8 @@ static void computeStats(CASDB &CAS, CASID TopLevel) {
   StringMap<ObjectKindInfo> Stats;
   ObjectKindInfo Totals;
   size_t NumGeneratedNames = 0;
-  Nodes[TopLevel].NumPaths = 1;
+  for (auto ID : TopLevels)
+    Nodes[ID].NumPaths = 1;
```

Comment:
(Maybe we should remove the "paths" column... not sure... not in this commit though)